content/creator/sdk7/3d-essentials/colliders.md

```diff
@@ -131,6 +131,10 @@ You can configure a `MeshCollider` component or the `GltfContainer` component to
 - `ColliderLayer.CL_CUSTOM1` through to `CL_CUSTOM8`: Can be used together with raycasts, so that a ray only detects collisions with one specific layer.
 - `ColliderLayer.CL_NONE`: Doesn't respond to collisions of any kind.
 
+{{< hint warning >}}
+**📔 Note**: To disable collisions form a `MeshCollider` component, delete the component. Do not set the collision layer to `ColliderLayer.CL_NONE`. There's a known issue with the `MeshCollider` component. Instead of disabling all collisions, it makes this value equivalent to the default (`ColliderLayer.CL_PHYSICS | ColliderLayer.CL_POINTER`).
```

Comment:
`ColliderLayer.CL_NONE` should work as expected

Reply:
We discussed this a bit over Slack but I'll bring some context here for the record:
I created an issue here: https://github.com/decentraland/documentation/issues/418
"The technical explanation: `CL_NONE` enum value is `0`, which is bitwise-treated the same as not having a `ColliderLayer` defined, which defaults to `CL_PHYSICS & CL_POINTER`."

A potential fix would be to change the enum value, but that would require more development time, in the mean time I think this warning is enough to be clear to creators that this is a known issue, what do you think @gonpombo8 ?